tools/common_lib/src/iumd_d3d12_impl.cpp

```diff
@@ -136,6 +142,16 @@ iumd::custom_metacommand::UmdD3d12Device::UmdD3d12Device(ID3D12Device* device, I
         sku_.threads_per_eu = 8;
         sku_.hw_simd_size = 16;
     }
+
+    switch(sku_.igfx)
+    {
+    case UMD_IGFX::eARLH:
+        sku_.l3_cache_size = 8192 * 1024;
```

Comment:
Hardcoded `8192` looks to be true, but ideally this should be computed:
l3_cache_size = l3_bank_count * l3_bank_size 
l3_bank_count  == DSS count 
l3_bank_size = 1024k bytes
OK to not change code to use this values, but please add a comment.
Also, what about values for non-ARL platforms?